exercises/perfect-numbers/example.rb

```diff
@@ -0,0 +1,39 @@
+class PerfectNumber
+  VERSION = 1
+  attr_accessor :num
+  
+  def initialize(num)
+    raise "#{num} is not a natural number." if num < 0
```

Comment:
Oh yeah I totally agree! I was confused at first because I thought you meant I should write a test to make sure num was getting set in initialize